### ISSUE TITLE
Missing data fields in WindowEventObj

### ISSUE THREAD

**Issue by pseudo-random** [nim-lang/sdl2#126]
[According to the SDL2 documentation](https://wiki.libsdl.org/SDL_WindowEvent), the SDL_WindowEvent (`WindowEventObj`) struct should contain two data fields called `data1` and `data2`. These fields are supposed to contain window event dependent data such as the new window size or position. The latest version of the sdl2 library does not provide access to the fields in question. It looks like they were removed in https://github.com/nim-lang/sdl2/commit/d4d6e6bda96f56bae31f3fa732208a73edba8eb7. Am I missing something here or were they simply removed by accident?

**Comment by Araq** [comment]
Indeed, seems to be an oversight. Ping @dom96

**Comment by dom96** [comment]
Yeah, I thought this was unused data and only wanted to fix the size of the object to make it castable. Feel free to fix it so the object is the same size and so it contains those fields.
